File: Typescript2/4.py
  
def ofchunk(arr,size):

    chunk = []

    for i in range(0,len(arr),size):

        chunk.append(arr[i:i+size])
    
    return chunk

arr =[1,2,3,4,5,6,7,8,9]
size = 3
print(ofchunk(arr,size))
    

File: Typescript2/4.ts

  
  var chunk = function(arr:number[] , size:number){

    const chunkarray:number[][] = []

    for ( let i=0 ; i< arr.length ; i += size){
        chunkarray.push(arr.slice(i,i+size))
    }

    return chunkarray
  }

   const arr = [1,2,3,4,5,6,7,8,9]
   const size = 3
   console.log(chunk(arr,size))